doc/source/style.rst

```diff
@@ -0,0 +1,25 @@
+.. _style:
+
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import numpy as np
+   import pandas as pd
+   np.set_printoptions(precision=4, suppress=True)
+   pd.options.display.max_rows = 15
+
+*******
+Styling
+*******
+
+.. versionadded:: 0.17.1
+
+
+You can apply **conditional formatting**, the visual styling of a DataFrame
+depending on the data within, by using the ``.style`` property.
+This is a property on every object that returns a ``Styler`` object, which has
+useful methods for formatting and displaying DataFrames.
+
```

Comment:
I realized that if you are going to render the notebook, then putting it in ``doc/source`` where you had it is prob correct. sorry. :)